### DISCUSSION TITLE
Sliderfollowcircle isnt animated

### DISCUSSION THREAD

**Discussion by enoslayd** [ppy/osu#16175] [Q&A]
In stable, ~~it pulsates with bpm (i think?)~~ it briefly expands when collecting slider ticks. (I cant provide any video/gif as i cant screen record)

**Comment by bdach** [comment]
https://github.com/ppy/osu/discussions/15331?

**Reply by enoslayd** [reply]
Omg its sliderfollowcircle not sliderb😅😅. Gonna edit the post

**Comment by bdach** [comment]
Example map where this is visible is https://osu.ppy.sh/beatmapsets/411435#osu/966177, at 03:41:645 (~~stolen~~ borrowed from stream chat).

This is similar to https://github.com/ppy/osu/pull/14832 in that the animations *are* incorrect but given that stable would animate the follow circle in various ways but then keep the actual input follow circle area constant I'm not sure whether that is something that should be brought back, and how to do it if so.
